commerce_coordinator/settings/base.py

```diff
@@ -90,16 +99,6 @@ def root(*path_fragments):
     'edx_rest_framework_extensions.auth.jwt.middleware.EnsureJWTAuthSettingsMiddleware',
 )
 
-# Cache Configuration
-CACHES = {
```

Comment:
Just to be clear, I removed this configuration for just letting you know about the culprit of the failed test cases. Ideally this configuration shouldn't be picked in the check run that was failing and instead it should use the test file. Also you added [duplicate Cache config](https://github.com/edx/commerce-coordinator/pull/155/files#diff-b2b59c815137417b558b63d9cdafd3965a4b443a0fc911938adcad6541499c7aR65) so you would want to keep just one of them after fixing the configuration file for the check run that was failing.
If we want to use `pymemcache` as a cache backend then we should install it as a dependency too and also make a ticket to configure it for stage and prod